docs/FAQ.md

```diff
@@ -0,0 +1,83 @@
+## Frequently Asked Questions
+
+Here are a few questions that have been asked often as well as answers from the community.
+
+If you have a question that is not answered here, feel free to ask in the Discord chat and suggest it to be placed here.
+
+Thanks!
+
+
+### Contents
+
+* [Will I make money with Zenbot?](#money)
+* [Is Docker necessary when installing Zenbot?](#docker)
+* [Why do simulations, paper trading, and live trading all yield different results?](#sim-live-differences)
+* [Why should I use simulations or paper trading if they do not reflect live trading?](#why-sim)
+* [Does Zenbot use Limit orders or Market orders?](#limit-market-orders)
+* [Why does Zenbot use Limit orders by default?](#limit-orders)
+* [Is there a community to get involved with Zenbot?](#community)
+* [Why do I keep getting a this "Nonce" error?](#error-1)
```

Comment:
I think this one is also often popping up:
`* [How do I update zenbot?]`

docker: `git pull && docker-compose down && docker rmi zenbot_server && docker-compose build && docker-compose up -d`
standalone: `git pull && npm install && ./zenbot.sh trade --paper`